livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -177,7 +178,7 @@ def __init__(
             AudioTranscription | InputAudioTranscription | None
         ] = NOT_GIVEN,
         input_audio_noise_reduction: NotGivenOr[
-            NoiseReductionType | InputAudioNoiseReduction | None
+            NoiseReductionType | NoiseReduction | None
```

Comment:
```suggestion
            NoiseReduction | InputAudioNoiseReduction | None
```
keep the `InputAudioNoiseReduction` for backward compatibility, I think it's also fine if we want to support 
 `NoiseReductionType | NoiseReduction | InputAudioNoiseReduction | None`

Reply:
@longcw it should be all good now, let me know if there's anything else!